pkgs/top-level/all-packages.nix

```diff
@@ -12864,7 +12864,18 @@ with pkgs;
 
   pipenv = callPackage ../development/tools/pipenv {};
 
-  pipewire = callPackage ../development/libraries/pipewire {};
+  pipewire =
+    let
+      # Break a dependency cycle
+      customSDL2 = SDL2.override {
+        pipewireSupport = false;
+      };
+    in callPackage ../development/libraries/pipewire {
+      SDL2 = customSDL2;
+      ffmpeg = ffmpeg.override {
```

Comment:
This is the small ffmpeg, right? How long does it take to build? I am not sure if this is the best solution we have.

Reply:
If by "small ffmpeg" you mean "not `ffmpeg-full`", then yes, this is the small ffmpeg. It takes 5 minutes and 4 seconds to build on my laptop. But I agree that this might not be the best solution.